addons/website/static/tests/tours/conditional_visibility.js

```diff
@@ -74,3 +94,54 @@ tour.register('conditional_visibility_2', {
     },
 },
 ]);
+
+tour.register("conditional_visibility_3", {
+    test: true,
+    url: "/",
+},
+[
+...wTourUtils.clickOnEditAndWaitEditMode(),
+checkEyeIcon("Text - Image", true),
+wTourUtils.dragNDrop(snippets[1]),
+// Click on the "Banner" snippet.
+wTourUtils.clickOnSnippet(snippets[1]),
+wTourUtils.changeOption("ConditionalVisibility", "we-toggler"),
+wTourUtils.changeOption("ConditionalVisibility", '[data-name="visibility_conditional"]'),
```

Comment:
I will create a task ASAP as `wTourUtils.changeOption("ConditionalVisibility", '[data-name="visibility_conditional"]'),` is working but `wTourUtils.changeOption("ConditionalVisibility", "[data-name='visibility_conditional']"),` is not.